packages/amplify-cli/src/__tests__/extensions/amplify-helpers/remove-resource.test.ts

```diff
@@ -3,7 +3,7 @@ import { stateManager, exitOnNextTick, ResourceDoesNotExistError, MissingParamet
 import * as inquirer from 'inquirer';
 import { updateBackendConfigAfterResourceRemove } from '../../../extensions/amplify-helpers/update-backend-config';
 import { removeResourceParameters } from '../../../extensions/amplify-helpers/envResourceParams';
-
+const path = require('path');
```

Comment:
Can we move this to the top of the file and use `import` like:

```js
import * as path from 'path';
// or
import { join } from 'path';
```

Reply:
Surely